lib/external/spokes.ts

```diff
@@ -48,19 +47,25 @@ export class SpokesClient {
             body: {
                 // The secret key for the Skill
                 uuid: this._secretKey,
-                // Should be the same as the UUID - also the secret key
-                diagnosticsKey: this._secretKey,
-                endpoint: {
+                diagnosticsKey: null,
+                endPoint: {
                     // The unique name/ID for the skill
                     name: this._id,
                 },
+                http: {
+                    url: `https://proxy.bespoken.tools?node-id=${this._secretKey}`,
```

Comment:
This should just be `https://proxy.bespoken.tools` instead of `https://proxy.bespoken.tools?node-id=${this._secretKey}`. That is why we are getting the an error message "Node id is not active: <UUID>,<UUID>"